src/resources/Activities/Activities.ts

```diff
@@ -33,6 +34,13 @@ export default class Activity extends Resource {
         );
     }
 
+    listAll(params?: ListActivitiesParams, activityFacet?: ActivityListingFilters) {
+        return this.api.post<PageModel<ActivityModel>>(
+            this.buildPath(`${Activity.getBaseUrlAllOrgs()}`, params),
```

Comment:
```suggestion
            this.buildPath(Activity.getBaseUrlAllOrgs(), params),
```

Reply:
added UT and applied the simplification! :)